### DISCUSSION TITLE
New Feature Idea

### DISCUSSION THREAD

**Discussion by AbdulBasit-MrRobo** [community/community#6875] [Copilot Conversations] [answered]
When suggesting all 10 suggestions in github copilot (shortcut : ctrl + enter) you can also add link to stackoverflow discussion too so one can get better understanding of solution provided by github copilot and can also share their thoughts too.

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Suggestions provided by GitHub copilot are created and generated by our AI. They're not coming from any source on the internet. As such we can't provide a link to any location.

**Reply by AbdulBasit-MrRobo** [reply]
Ooh I see. Thanks for solving doubt
